src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -259,10 +264,11 @@ def generate_parameters(self, parameter_id):
             # but it can cause deplicate parameter rarely
             total_params = self.get_suggestion(random_search=True)
         self.total_data[parameter_id] = total_params
+        self.running_data.append(parameter_id)
         params = _split_index(total_params)
         return params
 
-    def receive_trial_result(self, parameter_id, parameters, value):
+    def receive_trial_result(self, parameter_id, parameters, value, fake=False):
```

Comment:
the naming of "fake" is really confusing....

Reply:
Change to constant_liar